csrc/runtime/compiled_kernel.cpp

```diff
@@ -1259,18 +1259,6 @@ void CompiledKernel::compile(int64_t block_size) {
 
   kernel_code_ = codegen::generateCudaKernel(kernel, kernelName(), block_size);
 
-  // If NVFUSER_EXTERNAL_SRC is set, utilize the external source code.
```

Comment:
Moved to `getStructuredCode`, which is called in both `compile` and `recompileKernel`.